Cesium3DTilesSelection/include/Cesium3DTilesSelection/TilesetOptions.h

```diff
@@ -318,6 +319,13 @@ struct CESIUM3DTILESSELECTION_API TilesetOptions {
    * so it must be inexpensive to copy.
    */
   std::any rendererOptions;
+
+  /**
+   * @brief The ellipsoid to use for this tileset.
+   *
+   * If no ellipsoid is set, Ellipsoid::WGS84 will be used by default.
+   */
+  std::shared_ptr<CesiumGeospatial::Ellipsoid> ellipsoid = nullptr;
```

Comment:
Why is an Ellipsoid needed in both `TilesetOptions` and `TilesetExternals`? I think just one of these should be sufficient (probably `TilesetOptions`).

I agree with Brian's suggestion to avoid the shared_ptr, too. An embedded Ellipsoid value in the TilesetOptions should be just fine, and copy it where it's needed. I wouldn't suggest holding persistent references to this, though, just copy instead. For example, passing a reference to the ellipsoid in the TilesetOptions when calling some functions is 👍, but if that function then needs to keep the ellipsoid for later, it should do so by copy, not by storing the reference passed into the function. Ellipsoid is ~104 bytes, which isn't tiny, but not too prohibitive, either, as long as we're not storing it in every `Tile` or anything like that.